src/mrack/providers/podman.py

```diff
@@ -58,8 +58,33 @@ async def init(self, container_images, default_network, ssh_key, container_optio
         logger.info(f"{self.dsp_name}: Init duration {login_duration}")
 
     async def prepare_provisioning(self, reqs):
-        """Prepare provisioning."""
-        pass
+        """Pull missing images to prepare provisioning for podman."""
+        image_list = set()
+        image_list.add("quay.io/neoave/snappeas:fedora-33")
```

Comment:
Is having ` image_list.add("quay.io/neoave/snappeas:fedora-33")` here intentional? Looks as leftover from testing.

Reply:
Correct